.all-contributorsrc

```diff
@@ -247,6 +247,13 @@
       "contributions": [
         "bug"
       ]
+    },
+    {
+      "login": "Garbee",
+      "name": "Jonathan Garbee",
+      "avatar_url": "https://avatars.githubusercontent.com/u/868301?v=3",
+      "profile": "http://jonathan.garbee.me",
+      "contributions": []
```

Comment:
Don't forget to add your type of contribution by using `yarn add-contributor`, in this case it would be `code`, `test`, `doc`?